osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/DlmsHelperTest.java

```diff
@@ -248,6 +249,15 @@ public void testCorrectLogMessageForBitStringObject() {
         assertThat(logMessage).isEqualTo(expected);
     }
 
+    @Test
+    public void testByteArrayToHexString() throws ProtocolAdapterException {
+        byte[] bytes = new byte[] {25, 24, 7, 118};
+        DataObject dataObject = DataObject.newOctetStringData(bytes);
+        String hexString = dlmsHelper.readHexString(dataObject, "reading a Hexadecimal String");
+
+        assertThat(hexString).isEqualTo("19180776");
```

Comment:
Not formatted according to the conventions. Missing final modifiers, white space, `this.`-prefix.

Reply:
done